lib/streamlit/dataframe_util.py

```diff
@@ -436,6 +452,19 @@ def convert_anything_to_pandas_df(
             data = data.copy(deep=True)
         return data.to_series().to_frame()
 
+    if is_dask_object(data):
+        data = data.head(max_unevaluated_rows, compute=True)
+
+        if isinstance(data, (pd.Series, pd.Index)):
```

Comment:
how can `data` here be of type `pd.Series` or `pd.Index` when we are inside of the `is_dask_object` function?

Reply:
the `.head()` on a Dask object returns the pandas version of the dask object, either `DataFrame`, `Series`, or `Index`.

Reply:
I added a comment

Reply:
thank you!